lib/src/query_service.dart

```diff
@@ -114,8 +114,31 @@ class QueryService {
   }
 
   /// Generates query based on user inputs.
-  static String _generateQuery(String advertiserId, String insertionOrderId) {
-    return 'https://displayvideo.googleapis.com/v1/advertisers/$advertiserId/'
-        'insertionOrders/$insertionOrderId';
+  static RequestArgs _generateDV3Query(QueryType queryType,
+      String nextPageToken, String advertiserId, String insertionOrderId) {
+    switch (queryType) {
+      case QueryType.byAdvertiser:
+        {
```

Comment:
i think the inner brackets arent necessary?

Reply:
looks like these are still here

Reply:
Ohhh..I thought i can only get rid of brackets if the statement is one line.